persistence-jdbc/core/src/main/resources/play/reference-overrides.conf

```diff
@@ -1,17 +1,23 @@
 jdbc-journal.slick {
-  driver = ${jdbc-defaults.slick.driver}
+  profile = ${?jdbc-defaults.slick.profile}
```

Comment:
This and the a few other similar profile declaration below must use a question mark because users may have configured `slick.driver` instead. 

The goal is to let users react to the warnings that Slick will emit and switch to `slick.profile`. As soon as an user updates his configuration to use a profile, the driver settings will be ignored.